test/functional/users_controller_test.rb

```diff
@@ -5,6 +5,34 @@
 class UsersControllerTest < ActionController::TestCase
   include Devise::Test::ControllerHelpers
 
+  test '#show' do
+    sign_in users(:mockstar)
+    user = users(:jroes)
+
+    get :show, params: { id: user.id }
+
+    assert_redirected_to root_path
+  end
+
+  test '#update should update an user' do
+    user = users(:mockstar)
+    sign_in user
+
+    put :update, params: { user: { name: 'New Name' } }
+    user.reload
+
+    assert_redirected_to user
+    assert_equal 'New Name', user.name
+  end
+
+  test '#token_delete' do
+    sign_in users(:schneems)
+
+    get :token_delete, params: { account_delete_token: users(:schneems).account_delete_token }
+
+    assigns(:lonely_repos).any?
```

Comment:
You are missing an assertion here, otherwise great tests. Thanks!

Reply:
indeed,  sorry about that, I was new to automated tests back then heheh!